contrib/pylightning/lightning/plugin.py

```diff
@@ -1,10 +1,13 @@
-import sys
-import os
-import json
 import inspect
+import json
+import os
 import re
+import sys
 import traceback
 
+from .lightning import LightningRpc
+from os.path import join
```

Comment:
I prefer to import `os` or `os.path` since just importing `join` is a bit ambiguous as to what join operation this actually is. Also we already have `os` imported so why not just use that import?